lib/charms/glauth_k8s/v0/ldap.py

```diff
@@ -123,8 +123,10 @@ def _on_ldap_requested(self, event: LdapRequestedEvent) -> None:
 """
 
 from functools import wraps
+from string import Template
 from typing import Any, Callable, Literal, Optional, Union
 
+import ops
```

Comment:
I think you only need `from ops import Secret`

Reply:
oh, I can not do that because the naming will conflict with the class `Secret`.